cupyx/scipy/signal/_bsplines.py

```diff
@@ -37,3 +40,353 @@ def sepfir2d(input, hrow, hcol):
     filters = (hcol[::-1].conj(), hrow[::-1].conj())
     return cupyx.scipy.ndimage._filters._run_1d_correlates(
         input, (0, 1), lambda i: filters[i], None, 'reflect', 0)
+
+
+def _quadratic(x):
+    x = abs(cupy.asarray(x, dtype=float))
+    b = BSpline.basis_element(
+        cupy.asarray([-1.5, -0.5, 0.5, 1.5]), extrapolate=False)
+    out = b(x)
+    out[(x < -1.5) | (x > 1.5)] = 0
+    return out
+
+
+def _cubic(x):
+    x = cupy.asarray(x, dtype=float)
+    b = BSpline.basis_element(
+        cupy.asarray([-2, -1, 0, 1, 2]), extrapolate=False)
+    out = b(x)
+    out[(x < -2) | (x > 2)] = 0
+    return out
+
+
+def _coeff_smooth(lam):
```

Comment:
If we keep this computation on GPU, I'd like to make it fused.
```suggestion
@cupy.fuse()
def _coeff_smooth(lam):
```